src/pocketmine/Player.php

```diff
@@ -3026,6 +3027,18 @@ public function sendMessage($message){
 		*/
 	}
 
+	/**
+	 * Send a weather mode to aplayer.
+	 *
+	 * @param int $id
+	 */
+	public function sendWeather($id){
+		$pk = new LevelEventPacket();
+		$pk->evid = $this->getLevel()->getWeatherManager()->getEvid($id);
+		$pk->data = 90000; //Not sure if this is default.
```

Comment:
This is weather strength. Not sure on exact values, since this field is an int and weather strength is usually a float. Might want to check that.